quinn/src/send_stream.rs

```diff
@@ -138,7 +138,16 @@ impl SendStream {
         Finish { stream: self }.await
     }
 
-    #[doc(hidden)]
+    /// Attempt to shut down the send stream gracefully.
+    ///
+    /// No new data may be written after calling this method. Completes when the peer has
+    /// acknowledged all sent data, retransmitting data as needed.
+    ///
+    /// On success, return Poll::Ready(Ok(())).
+    ///
+    /// If the stream is not ready for writing, the method returns Poll::Pending and arranges
+    /// for the current task (via cx.waker().wake_by_ref()) to receive a notification when the
+    /// stream becomes writable or is closed.
```

Comment:
This paragraph is confusing. Perhaps it was copy-pasted in error from `poll_write`? I think the first two paragraphs are sufficient documentation.

Reply:
Indeed it was.